config/locales/en.controllers.yml

```diff
@@ -132,6 +132,8 @@ en:
         Please add it again to make a reservation.
       update:
         failure: The reservation cannot be updated.
+      switch_instrument:
+        prior_is_still_running: Cannot "Begin Reservation" when a previously scheduled reservation is ongoing.
```

Comment:
The quoted `“Begin Reservation”` here seems strange — maybe consider something like:

> Could not begin a reservation because a previously-scheduled one is still ongoing.

Reply:
I don't disagree it's a little funky but it's the text they asked for.